tests/unit/amo/middleware/test_cspMiddleware.js

```diff
@@ -39,11 +39,12 @@ describe(__filename, () => {
         // eslint-disable-next-line global-require
         const config = require('config');
         const cspConfig = config.get('CSP').directives;
-        expect(cspConfig.scriptSrc).toContain(cdnHost);
```

Comment:
Some of these could be inverted (with a comment), in-case someone looks to 'optimize' the CSP back to a single origin.

Reply:
What should be inverted?

Reply:
The test expectation. To cover disallowing an origin without a path.